CommunicationCallingXamarinSampleApp/CommunicationCallingXamarinSampleApp.Android/Composite.cs

```diff
@@ -167,11 +208,11 @@ public void Finalized()
 
             public void Handle(Java.Lang.Object eventArgs)
             {
-                if (eventArgs is CallCompositeCallStateEvent)
-                {
-                    var callState = eventArgs as CallCompositeCallStateEvent;
-                    Console.WriteLine(callState.Code.ToString());
-                }
+                //if (eventArgs is CallCompositeCallStateEvent)
```

Comment:
CommentedOut these lines, as 1.4.0 aar did not include CallCompositeCallStateEvent class.
@iaulakh